go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/go-openapi/strfmt v0.19.5
 	github.com/google/go-cmp v0.5.2
 	github.com/google/go-containerregistry v0.1.2
-	github.com/google/go-github/v28 v28.1.1
+	github.com/google/go-github/v32 v32.1.0
```

Comment:
Was there code added to care with the *upload api issue* in v32? If yes, it this marked with a `TODO` to remove in the future?

Reply:
It is resolved without another todo in `github.go` ->we now avoid using a convenience client ...